Wikipedia/Code/WMFImageController.swift

```diff
@@ -454,6 +454,7 @@ extension WMFImageController {
             }
 
             let success = { (didCache: Bool) in
+                self.imageManager.wmf_removeImageForURL(imageURL, fromDisk: false)
```

Comment:
Although... if we're not using the mem cache is there any point in using SDWebImage ;)